docs/src/main/sphinx/release/release-432.md

```diff
@@ -0,0 +1,57 @@
+# Release 432 (2 Nov 2023)
+
+:::{note}
```

Comment:
Should we add an anchor to this and link all icons to it? Or at least be able to link to it later from other release notes? Or even have this documented elswhere? How do readers otherwise know what the symbol is for?

Reply:
My opinion is:

- Keep this note here to indicate that this is the first release where this paradigm was implemented.
- Add docs elsewhere.
- Have the warning emoji link to those docs moving forwards.